images/linux/scripts/tests/Tools.Tests.ps1

```diff
@@ -284,7 +284,7 @@ Describe "Kubernetes tools" {
     }
 
     It "kubectl" {
-        "kubectl version" | Should -MatchCommandOutput "Client Version: v"
+        "kubectl version --client" | Should -MatchCommandOutput "Client Version: v"
```

Comment:
I just saw it in the window script: https://github.com/actions/runner-images/blob/main/images/win/scripts/Tests/Tools.Tests.ps1#L98

We could also change it to `"kubectl version --client=true"`, if you want it to be the same.